### PR TITLE
marc21: holdings and authority rules

### PR DESCRIPTION
**Pull request by jirikuncar**
Signed-off-by: Jiri Kuncar jiri.kuncar@cern.ch

### CODE REVIEW

**Comment by jirikuncar** [comment]
ping @aw-bib

**Comment by jirikuncar** [comment]
@mvesper @aw-bib can you please check these rules and provide test records?

File: dojson/contrib/marc21/__init__.py [hunk 1]
```diff
@@ -7,6 +7,10 @@
 # modify it under the terms of the Revised BSD License; see LICENSE
 # file for more details.
 
-from model import marc21
+"""MARC standards based on `www.loc.gov/marc/ <http://www.loc.gov/marc/>`_."""
 
-__all__ = ('marc21',)
+from __future__ import absolute_import
+
+from .model import marc21, marc21_holdings
```
Comment (egabancho) [inline dojson/contrib/marc21/__init__.py]: should we go for the same naming convention for all of them `marc21_bibliographic`, `marc21_holdings`, `marc21_authority`, etc.?

Comment (jirikuncar) [inline dojson/contrib/marc21/__init__.py]: We can add an alias for `marc21_bibliographic` but it's not part of this PR.

**Comment by aw-bib** [comment]
ping @fjorba I guess you have some good test cases as well.

**Comment by tiborsimko** [comment]
@jirikuncar Can you please rebase against latest master?  We're talking with @Kennethhole who has a nice set of MARC21 authority records that we can use for verification...

**Comment by jirikuncar** [comment]
@tiborsimko @Kennethhole :baseball:

**Comment by tiborsimko** [comment]
Testing data from @Kennethhole... The following is required for authority records to work:

``` console
$ cd dojson/contrib/marc21/fields
$ perl -pi -e 's,\@marc21\.,\@marc21_authority\.,g' ad*.py
```

**Comment by jirikuncar** [comment]
@tiborsimko thanks. I have fixed it.

**Comment by jirikuncar** [comment]
@tiborsimko fixed `value[0]`

**Comment by tiborsimko** [comment]
Here is the list of missing rules for @Kennethhole's authority demo files:
- 090__
- 099__
- 10020
- 1002_
- 40020
- 4002_
- 460_0
- 590__
- 600__
- 627__
- 647__
- 670_0
- 670__
- 675__
- 6780_
- 6781_
- 678__
- 690__
- 699__
- 949__

@Kennethhole Can you please check which of these are relevant?  E.g. the field 400 indicator 1 cannot have a value of 2 according to the specs... http://www.loc.gov/marc/authority/ad400.html

**Comment by jirikuncar** [comment]
- ~~090__~~ local
- ~~099__~~ local
- ~~10020~~ invalid indicator
- ~~1002_~~ invalid indicator
- ~~40020~~ invalid indicator
- ~~4002_~~ invalid indicator
- ~~460_0~~ undefined field
- ~~590__~~ undefined field
- ~~600__~~ undefined field
- ~~627__~~ undefined field
- ~~647__~~ undefined field (http://www.loc.gov/marc/authority/ad64x.html)
- 670_0 FIXED
- 670__ FIXED
- 675__ FIXED (Q: shall we join 670 and 675 as source_data?)
- 6780_ FIXED
- 6781_ FIXED
- 678__ FIXED
- ~~690__~~ local
- ~~699__~~ local
- ~~949__~~ local

**Comment by tiborsimko** [comment]
@jirikuncar Thanks, LGTM
